### PR TITLE
WSIO-132 Service for contact form

### PR DESCRIPTION
**Pull request by andrechern**
Contact us component was changed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

**Review by Rafal-DS — COMMENTED** [review]
@andrechern later there will be quarkus service, that actually sends the email, right?

**Comment by andrechern** [comment]
Yes

> @andrechern later there will be quarkus service, that actually sends the email, right?

yes

File: application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html [hunk 1]
```diff
@@ -17,39 +17,51 @@
 */-->
 
 <sly data-sly-use.model="pl.ds.bulma.components.models.ContactFormComponent">
-  <form>
+  <form data-config-captcha-public-key='${model.captchaPublicKey}'
+        data-config-http-end-point='${model.configEndpoint}'>
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html:21]: We have this configuration added to the body: https://github.com/websight-io/bulma/blob/main/application/backend/src/main/resources/libs/bulma/components/page/body.html#L23

We can remove it from the body, but we need to update js files as well to use both (recaptcha and endpoint) configs from the form definition

File: application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html [hunk 2]
```diff
@@ -17,39 +17,51 @@
...
-        <div class="control has-icons-left">
-          <input class="input" type="tel" name="phone" placeholder="Phone" required>
-          <span class="icon is-small is-left"><i class="fas fa-phone"></i></span>
+
+    <div class="columns is-desktop">
+      <div class="column">
+        <div class="field">
+          <label class="label">Full Name</label>
+          <div class="control">
+            <input class="input" type="text" name="subject" placeholder="Full Name" required>
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html:44]: It has incorrect name

Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/contactform/contactform.html]: It has incorrect name

**Review by paweljozwicki — CHANGES_REQUESTED** [review]

**Pull request merged**